cmd/apps/istio_app.go

```diff
@@ -114,11 +114,21 @@ func MakeInstallIstio() *cobra.Command {
 		outputPath := path.Join(chartPath, "istio")
 
 		if initIstio, _ := command.Flags().GetBool("init"); initIstio {
-			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, wait)
+			// Waiting for the crds to appear
+			err = helm3Upgrade(outputPath, "istio/istio-init", namespace, "", defaultVersion, overrides, true)
```

Comment:
Please use the wait flag instead of forcing wait to be true.

I'm OK if you want to add a flag and set it to default a true value. See the other apps for examples or ask in #arkade on Slack if you need help with that.

Reply:
If I don´t force and to wait, the second step will always fail. I think its a bad behavior.